ReactSkia/core_modules/RSkAlertManager.cpp

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include "RSkAlertManager.h"
+
+#include "sdk/sdkStylesConfig.h"
+
+namespace facebook {
+namespace react {
+
+std::mutex lockMsgDrawing;
+
+#define FONT_SIZE 24
+#define LINE_SPACE 14
+#define OFFSET_VERTICAL 30
+
+RSkAlertManager::RSkAlertManager(
+    const std::string &name,
+    std::shared_ptr<facebook::react::CallInvoker> jsInvoker,
+    facebook::react::Instance *bridgeInstance)
+    : TurboModule(name, jsInvoker) {
+  methodMap_["alertWithArgs"] = MethodMetadata{1, alertWithArgsWrapper};
+}
+
+void RSkAlertManager::processAlertMessages(std::shared_ptr<Alert> alertPtr) {
+  alertPropsList_.push_back(alertPtr);
+  if (ALERT_WINDOW_DESTRUCT == alertWindowState_) {
+    alertWindowState_ = ALERT_WINDOW_CREATE;
+    createAlertWindow();
+  } else {
+    if (ALERT_WINDOW_ACTIVE == alertWindowState_) {
+      drawAlertMsg();
+      commitDrawCall();
+    }
+  }
+}
+
+facebook::jsi::Value RSkAlertManager::alertWithArgsWrapper(
+    facebook::jsi::Runtime &rt,
+    TurboModule &turboModule,
+    const facebook::jsi::Value *args,
+    size_t count) {
+  auto &alertManagerInstance = static_cast<RSkAlertManager &>(turboModule);
+  auto alertProps = facebook::jsi::dynamicFromValue(rt, args[0]);
+  alertManagerInstance.processAlertMessages(std::make_shared<Alert>(
+      alertProps["title"].getString(), alertProps["message"].getString()));
```

Comment:
"message" object can be either message string or undefined as per definition in Alert.js
So should we check before calling getString()